src/explore-education-statistics-admin/src/pages/release/content/components/BasicReleaseSummary.tsx

```diff
@@ -20,7 +20,7 @@ interface ReleaseTypeIcon {
 }
 
 const nationalStatisticsLogo: ReleaseTypeIcon = {
-  url: '/assets/images/UKSA-quality-mark.jpg',
+  url: '/assets/images/UKSA-quality-mark2.jpg',
```

Comment:
Seems to me like there'd be no benefit in having the original mark.jpg image. Maybe we can just replace it completely with mark2.jpg so we only have the one?